docs/geedocs/5.3.2/answer/HostnameFqdnConfiguration.html

```diff
@@ -0,0 +1,53 @@
+<title>Hostname and Fully Qualified Domain Name Configuration</title>
+<link rel="stylesheet" href="../css/styles.css" type="text/css" media="all" />
+<link rel="stylesheet" href="../css/containers.css" type="text/css" media="all" /> </head> <body>
+<!-- HostnameFqdnConfiguration.html -->
+<div class="container">
+  <div class="sidebar1">
+<p class="sidebar-toc"><a href="../answer/4492598.html">Installation overview</a></p>
+<p class="sidebar-toc"><a href="../answer/4492599.html">System Requirements for Fusion and GEE Server</a></p>
+<p class="sidebar-toc"> --> <a href="" class="current-file">Hostname and Fully Qualified Domain Name Configuration</a></p>
+<p class="sidebar-toc"><a href="../answer/4492484.html">Before you install</a></p>
+<p class="sidebar-toc"><a href="../answer/4492604.html">Installing</a></p>
+<p class="sidebar-toc"><a href="../answer/4492616.html">Verifying and troubleshooting</a></p>
+<p class="sidebar-toc"><a href="../answer/4492617.html">Uninstalling</a></p>
+  </div>
+  <div class="content">
+<a name="top_of_file"></a>
+<p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
+<h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Install Google Earth Enterprise</h1>
+<h2>Hostname and Fully Qualified Domain Name Configuration</h2>
+<p>In order to make sure that the Apache server correctly determines the Fully Qualified Domain Name (FQDN) of the
+server, and that the virtual hosts use the correct server URL the hostname should be configured correctly. This is
+preferred instead of modifying the Apache server name configuration so that all the software looks to the same
+source for determining FQDNs. If the command <code>hostname -f</code> returns the correct FQDN hostname then
+the system is correctly configured to run Open GEE. If not the following instructions may help.</p>
+<p>Ideally the network where GEE is installed has a DNS set up to resolve hostnames so that IP traffic can be
+routed appropriately. However in cases where there is no DNS, static hostnames can be used as well. With static
+hostnames the <code>hostname</code> or <code>hostnamectl</code> command should be used to set the hostname to its
+FQDN. The <code>/etc/hostname</code> file should be updated to match as well so the name stays after reboots. If
+setting the hostname to the FQDN is not possible then setting the FQDN in the <code>/etc/hosts</code> file is
+necessary. When using a static hostname some operating systems may still give a warning on Apache startup even if
+it correctly determined the FQDN. Adding the FQDN to <code>/etc/hosts</code> would fix this as well. If adding the
+FQDN to <code>/etc/hosts</code> is needed, then the FQDN followed by just the host portion should be added to any
+line that contains the IP address the server will be listening on, like this: </p>
+<p><code>127.0.0.1	myserver.mydomainname.com myserver localhost</code></p>
```

Comment:
You should add a sentence specifying that this line with the hostname should be the first line of the file